src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -86,13 +91,12 @@ case class BaggedSingleResult(
 
   private lazy val singleScores: Vector[Double] = {
     // Compute the variance of the ensemble of predicted values and divide by the size of the ensemble an extra time
-    val varT = 1.0 / (treePredictions.size * treePredictions.size) * treePredictions.map(y => Math.pow(y - expected, 2.0)).sum
+    val varT = treeVariance / treePredictions.length
```

Comment:
Why divide by ensemble size again?

Reply:
We're going to subtract the correction from each term in a sum of size `treePredictions.length`, so this division accounts for that subsequent sum.

If we were only computing the variance, then the correction would just be subtracted at the end.  We do it term-by-term to support the `getImportanceScores` and `getInfluenceScores` methods.